src/styles/blocks/stars1.css

```diff
@@ -0,0 +1,11 @@
+.stars--1 :nth-child(-n + 1)::after {
+  content: "";
+  position: absolute;
+  background-image: url(/images/star-active.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  width: var(--width-star);
+  height: var(--height-star);
+  flex-shrink: 0;
+  margin-right: var(--horizontal-gap);
+}
```

Comment:
.stars--1 .stars__star:nth-child(-n + 1),
.stars--2 .stars__star:nth-child(-n + 2),
.stars--3 .stars__star:nth-child(-n + 3),
.stars--4 .stars__star:nth-child(-n + 4),
.stars--5 .stars__star:nth-child(-n + 5) {
  background-image: url("images/star-active.svg");
}


And you don't need to create a separate file for each block with stars, because they are the same